src/Sylius/Bundle/AddressingBundle/Form/Type/CountryChoiceType.php

```diff
@@ -41,14 +41,24 @@ public function configureOptions(OptionsResolver $resolver): void
             ->setDefaults([
                 'choice_filter' => null,
                 'choices' => function (Options $options): iterable {
-                    if (null === $options['enabled']) {
-                        $countries = $this->countryRepository->findAll();
-                    } else {
+                    if (null !== $options['enabled'] && $options['multiple']) {
```

Comment:
Instead of changing the logic here by adding a condition with `multiple` here, you can probably set the proper value of `enabled` in the appropriate form type